inst/@sym/horzcat.m

```diff
@@ -62,7 +62,7 @@
           'return sp.Matrix.hstack(*_proc),'
           };
 
-  varargin = sym(varargin);
+  varargin = cell_array_to_sym(varargin);
```

Comment:
Sure, it sounds better to be that way. But again, we have some odd conversions in place which might make it difficult (or rather would required many conditionals)....
Thanks for the debug.. I actually tried to debug it with breakpoints but the breakpoints could not be put on a classdef class.... I just read on the wiki that breakpoints via gui are not yet supported for classdef and have to be put via the cli....